extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -28,15 +30,65 @@ class AmpStickyAd extends AMP.BaseElement {
   isLayoutSupported(layout) {
     return layout == Layout.NODISPLAY;
   }
-
   /** @override */
   buildCallback() {
     /** @const @private {boolean} */
     this.isExperimentOn_ = isExperimentOn(this.getWin(), TAG);
+    this.viewport_ = this.getViewport();
     if (!this.isExperimentOn_) {
       dev.warn(TAG, `TAG ${TAG} disabled`);
       return;
     }
+    this.viewport_ = this.getViewport();
```

Comment:
Is this the latest version? I'm still not seeing jsdocs for `this.*` variables...